acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/GithubCreationPage.java

```diff
@@ -131,10 +117,12 @@ public void beginCreationFlow(String jobName) throws IOException {
     }
 
     public void completeCreationFlow(String apiKey, String org, String pipeline, boolean createJenkinsFile) {
-        if(wait.until(wait.orVisible(
-            driver -> apiKeyInput,
-            driver -> driver.findElement(getOrgSelector(org)))) == 1) {
-
+        if(wait.until(
```

Comment:
howcome we have to nest these two things called "wait"?

Reply:
The "wait" object seems to contain a bunch of dom related utils. It probably should be refactored to waitUntilOneIsVisible, but i honestly don't know where else its being used. orVisible returns 0 or 1 depending on which path is true.

Its kinda gross, but I rather save it to another pass